src/Component.php

```diff
@@ -99,6 +99,10 @@ protected function determineDefaultTag(): string
 
         $tag = Str::kebab(end($baseComponentName));
 
+        if (Str::contains($tag, '_')) {
```

Comment:
I think we can drop this check as string replace wont replace anything if there's no `_` to replace in the first place.

Reply:
I agree with @AlexVanderbist - this check only slows the method down.